src/lib/openapi/mappers/strategy.mapper.ts

```diff
@@ -0,0 +1,35 @@
+import { Mapper } from './mapper';
+import { IStrategyConfig } from '../../types/model';
+import { StrategySchema } from '../spec/strategy-schema';
+import { CreateStrategySchema } from '../spec/create-strategy-schema';
+import { UpdateStrategySchema } from '../spec/update-strategy-schema';
+
+export class StrategyMapper
+    implements
+        Mapper<
+            StrategySchema,
+            IStrategyConfig,
+            CreateStrategySchema | UpdateStrategySchema
+        >
+{
+    fromPublic(input: StrategySchema): IStrategyConfig {
+        return {
+            ...input,
+        };
+    }
+
+    toPublic(input: IStrategyConfig): StrategySchema {
+        return {
+            ...input,
+        };
+    }
```

Comment:
Maybe Mapper could have these no-ops as default implementations for fromPublic/toPublic.

Reply:
it looks like a noop but it handles the optional/not-optional type errors